build.cmd

```diff
@@ -204,6 +204,19 @@ call                                 "%__VSToolsRoot%\VsDevCmd.bat"
 
 REM =========================================================================================
 REM ===
+REM === Restore optimization profile data
+REM ===
+REM =========================================================================================
+
+echo %__MsgPrefix%Restoring the OptimizationData Package
```

Comment:
I am not sure I understand what would be the problem with adding that option. By default, it would be on and if I want to skip that, I'd pass an option to do that. We already have a bunch of options to skip various parts of the build (skipconfigure, skipmscorlib, skipnative, skiptests, skipbuildpackages), so this would be just another option of that kind.
Moreover, for the Unix build, we need to be able to run the build.sh on new platforms where we cannot run any managed code yet. So any attempt to restore something needs to be optional over there.